vector/src/main/java/im/vector/app/features/home/room/detail/search/SearchResultItem.kt

```diff
@@ -66,6 +67,7 @@ abstract class SearchResultItem : VectorEpoxyModel<SearchResultItem.Holder>() {
                     val displayName = it.threadSummarySenderInfo?.displayName
                     val avatarUrl = it.threadSummarySenderInfo?.avatarUrl
                     avatarRenderer.render(MatrixItem.UserItem(userId, displayName, avatarUrl), holder.threadSummaryAvatarImageView)
+                    holder.threadSummaryConstraintLayout.onClick(threadSummaryListener)
```

Comment:
This is not new, but the id `threadSummaryConstraintLayout` should be renamed to something more generic like `threadSummaryLayout` or `threadSummaryContainer`. Applicable to some other view ids, and not a blocker.

Reply:
I updated `searchThreadSummaryConstraintLayout` to `searchThreadSummaryContainer`, here might be good to have something generic for future changes. However, sometimes it is useful to have the exact type (ConstraintLayout, TextView, ImageView, etc ) in the id while the developer knows at once the type.